docusaurus/docs/reactnative/guides/debug_mode_using_flipper_plugin.mdx

```diff
@@ -0,0 +1,72 @@
+---
+id: debug-mode-using-flipper-plugin
+sidebar_position: 2
+title: Debug Stream Chat Application using Flipper plugin
+---
+
+## Overview
+
+Flipper is a platform for debugging iOS, Android and React Native apps. You can debug your application having Stream Chat in Flipper using our interface within the Flipper Desktop app. This will help you analyze multiple elements of the application which are a part of Stream Chat React Native interface.
```

Comment:
```suggestion
Flipper is a platform for debugging iOS, Android and React Native apps. You can debug Stream Chat within your application by using our interface within the Flipper Desktop app. This will help you analyze multiple parameters of the application which are a part of Stream Chat React Native interface.
```